### PR TITLE
improvement: block closing tickets with unclosed events when phpui.bl…

### PR DESCRIPTION
**Pull request by interduo**
…ockt_closing_tickets_with_unclosed_assigned_events is set (resolves partly #564)

### CODE REVIEW

File: doc/ChangeLog [hunk 1]
```diff
@@ -55,6 +55,8 @@ version ?.??.?? (????-??-??):
     in invoice report [chilan]
   - improvement: allow to change ticket state from RT_NEW to RT_OPEN when variable
     phpui.helpdesk_allow_change_ticket_state_from_open_to_new is set [interduo]
+  - improvement: block closing tickets with unclosed events when
+    phpui.blockt_closing_tickets_with_unclosed_assigned_events is set [interduo]
```
Comment (chilek) [inline doc/ChangeLog]: phpui.helpdesk_block_ticket_close_with_open_events?

File: lib/LMSManagers/LMSHelpdeskManager.php [hunk 2]
```diff
@@ -658,7 +658,7 @@ public function GetTicketContents($id)
 				o.name AS ownername, t.createtime, t.resolvetime, t.subject, t.deleted, t.deltime, t.deluserid,
 				t.address_id, va.location, t.nodeid, n.name AS node_name, n.location AS node_location,
 				t.netnodeid, nn.name AS netnode_name, t.netdevid, nd.name AS netdev_name,
-				t.verifierid, e.name AS verifier_username, t.deadline
+				t.verifierid, e.name AS verifier_username, t.deadline, unclosedeventscount
```
Comment (chilek) [inline lib/LMSManagers/LMSHelpdeskManager.php]: openeventcount

File: lib/LMSManagers/LMSHelpdeskManager.php [hunk 3]
```diff
@@ -670,6 +670,10 @@ public function GetTicketContents($id)
 				LEFT JOIN netnodes nn ON nn.id = t.netnodeid
 				LEFT JOIN netdevices nd ON nd.id = t.netdevid
 				LEFT JOIN vusers e ON (t.verifierid = e.id)
+				LEFT JOIN (
+					SELECT SUM(CASE WHEN closed != 2 THEN 1 ELSE 0 END) AS unclosedeventscount,
```
Comment (chilek) [inline lib/LMSManagers/LMSHelpdeskManager.php]: openeventcount

File: lib/locale/pl/strings.php [hunk 4]
```diff
@@ -26,6 +26,8 @@
 
 $_LANG['<!rt>hours'] = 'godzin';
 $_LANG['Network node location:'] = 'Lokalizacja węzła sieciowego:';
+$_LANG['Ticket have unclosed assigned events!'] = 'Zgłoszenie posiada niezamknięte zdarzenia!';
```
Comment (chilek) [inline lib/locale/pl/strings.php]: open

File: lib/locale/pl/strings.php [hunk 5]
```diff
@@ -26,6 +26,8 @@
...
+$_LANG['Unclosed assigned events count:'] = 'Ilość przypisanych niezamkniętych zdarzeń:';
```
Comment (chilek) [inline lib/locale/pl/strings.php]: raczej: event count, a nie events count

File: doc/ChangeLog (hunk 1 above)
Comment (interduo) [reply doc/ChangeLog]: ok zmienię

Comment (interduo) [reply doc/ChangeLog]: ticket czy tickets?

**Pull request merged**